CITATIONS.md

```diff
@@ -10,33 +10,33 @@
 
 ## Pipeline tools
 
-- [QUILT](https://pubmed.ncbi.nlm.nih.gov/34083788/)
+- [bcftools](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3198575/)
 
-> Davies, R. W., Kucka, M., Su, D., Shi, S., Flanagan, M., Cunniff, C. M., ... & Myers, S. (2021). Rapid genotype imputation from sequence with reference panels. Nature genetics, 53(7), 1104-1111.
+  > Li, H. (2011). A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics, 27(21), 2987-2993.
```

Comment:
In the Pipeline tools the ">" is preceded by a white space but not in the software part.
Could you choose one ?